src/main/java/com/documaster/rms/noark/ws/noarkentities/MappeBase.java

```diff
@@ -230,6 +231,16 @@ public void setMappetype(Mappetype mappetype) {
 		return link(MERKNAD_LINK, merknad);
 	}
 
+	public LinkAction<TEntity> linkNasjonalIdentifikator(String... nasjonalIdentifikatorIds) {
+
+		return link(NASJONAL_IDENTIFIKATOR_LINK, nasjonalIdentifikatorIds);
+	}
+
+	public LinkAction<TEntity> linkNasjonalIdentifikator(NasjonalIdentifikator... nasjonalIdentifikator) {
```

Comment:
I think this can become:
```suggestion
	public LinkAction<TEntity> linkNasjonalIdentifikator(NasjonalIdentifikator<?>... nasjonalIdentifikator) {
```

Reply:
Addressed with [this commit](https://github.com/documaster/ws-client-n5-java/pull/16/commits/b3ed4002853a3f25c3ff0b722ef0640c6eeea787).